### ISSUE TITLE
Resolver tarefas para complementar a resolução do desafio "Matrizes"

### ISSUE THREAD

**Issue by rafaelTsu** [EngSoft-UFMS/algoritmos_e_programacao_1#33]
Resolver as seguintes tasks no código a partir da função cria_matriz:
- [ ] Calcular soma ou média de uma linha específica da matriz;
- [ ] Calcular soma ou média de uma coluna específica da matriz;
- [ ] Calcular soma ou média da diagonal principal;
- [ ] Calcular soma ou média da diagonal secundária;
@wagnerUFMS pode resolver essas tarefas?

**Comment by wagnerUFMS** [comment]
Preciso criar funções para resolver essas tarefas? Ou somente para criar a matriz e imprimir?

**Comment by rafaelTsu** [comment]
Sim, se possível realizar essas operações implementando suas próprias funções.
